### ISSUE TITLE
Update for WAI-ARIA 1.1

### ISSUE THREAD

**Issue by kyoshino** [bzdeck/bzdeck#397]
See the [change log](https://www.w3.org/TR/wai-aria-1.1/#changelog). Our complete support should be done in #14.
- New: `feed`, `cell`, `table` roles
- Changed: ...

**Comment by kyoshino** [comment]
I'll revisit this when working on #14.
